storage/remote/otlptranslator/prometheusremotewrite/helper_test.go

```diff
@@ -531,7 +531,7 @@ func TestCreateAttributes(t *testing.T) {
 				}),
 				PromoteScopeMetadata: tc.promoteScope,
 			}
-			lbls, err := createAttributes(resource, attrs, tc.scope, settings, tc.ignoreAttrs, false, prompb.MetricMetadata{}, model.MetricNameLabel, "test_metric")
+			lbls, err := createAttributes(resource, attrs, tc.scope, settings, tc.ignoreAttrs, false, prompb.MetricMetadata{}, 0, model.MetricNameLabel, "test_metric")
```

Comment:
[Nit] Could we use the corresponding constant for improved readability, and not rely on the constant's value?
```suggestion
			lbls, err := createAttributes(resource, attrs, tc.scope, settings, tc.ignoreAttrs, false, prompb.MetricMetadata{}, pmetric.AggregationTemporalityUnspecified, model.MetricNameLabel, "test_metric")

```